tests/lib/Mail/MailerTest.php

```diff
@@ -102,7 +102,7 @@ public function mailAddressProvider(): array {
 			['lukas@éxämplè.com', true],
 			['españa@domain.com', true],
 			['asdf', false],
-			['lukas@owncloud.org@owncloud.com', false]
+			['lukas@owncloud.com@owncloud.com', false]
```

Comment:
```suggestion
			['lukas@domain.com@owncloud.com', false]
```
This one might look odd to future readers? They might wonder if this is testing a special case of repeating the same domain twicce?

Reply:
:heart: